tests/phpunit/Unit/Connection/CallbackConnectionProviderTest.php

```diff
@@ -0,0 +1,48 @@
+<?php
+
+namespace SMW\Tests\Connection;
+
+use SMW\Connection\CallbackConnectionProvider;
+
+/**
+ * @covers \SMW\Connection\CallbackConnectionProvider
+ * @group semantic-mediawiki
+ *
+ * @license GNU GPL v2+
+ * @since 3.0
+ *
+ * @author mwjames
+ */
+class CallbackConnectionProviderTest extends \PHPUnit_Framework_TestCase {
+
+	public function testCanConstruct() {
+
+		$callback = function() {};
+
+		$this->assertInstanceOf(
+			CallbackConnectionProvider::class,
+			new CallbackConnectionProvider( $callback )
+		);
+	}
```

Comment:
IMO this does not add value and is thus harmful clutter. (Yes I realize I introduced the first tests like this in the codebase, this was a mistake)

Reply:
It works for me!

Reply:
> harmful clutter

Disagree, this allows to check whether the constructor does work or not.